zkevm-circuits/src/keccak_circuit/keccak_bit.rs

```diff
@@ -607,8 +607,9 @@ impl<F: Field> KeccakBitConfig<F> {
         layouter: &mut impl Layouter<F>,
         inputs: &[Vec<u8>],
         r: F,
+        capacity: Option<usize>,
```

Comment:
The capacity units is number of permutations / keccak_f's that the circuit can handle right?

Reply:
That's right. Would you prefer a different name?